static/app/views/replays/types.tsx

```diff
@@ -170,7 +170,7 @@ export interface ReplayError {
 
 export type DeadRageSelectorItem = {
   aria_label: string;
-  dom_element: string;
+  dom_element: {fullSelector: string; projectId: number; selector: string};
   element: string;
```

Comment:
should this be `string` still?
```suggestion
  element: ReplayClickElement;
```

or just `element: Element`

Reply:
oh, sry nvm

Reply:
we don't even need `element` i totally forgot to remove it